src/components/ContactUsForm/encodeFormData.jsx

```diff
@@ -0,0 +1,6 @@
+// Converts object to proper form data(string)
+export const encodeFormData = (data) => {
```

Comment:
actually it's cleaner to put this in the same file as.the component. under the imports. it's harder for readability when one function is in its own file because people need to jump files to read it

Reply:
I moved the function under the component file. PTAL